src/test/regress/bin/normalize.sed

```diff
@@ -76,7 +76,7 @@ s/ *$//g
 
 # pg12 changes
 s/Partitioned table "/Table "/g
-s/\) TABLESPACE pg_default$/\)/g
+s/\) TABLESPACE pg_default/\)/g
```

Comment:
what is the reason for this change?

Reply:
Error log was differing in pg11 and pg12 in one of the outputs (that I cannot remember now), and we preferred to normalize the output as it was easier to achieve that difference with this one-liner normalization change